resume.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
```

Comment:
your resume page looks great on mobile, but looks to be a bit broken on desktop size